### ISSUE TITLE
Input sensors

### ISSUE THREAD

**Issue by cdcseacave** [WU-CVGL/MBA-SLAM#1]
Great paper, and amazing results.
Did you also test how your algorithm works with lower quality RGBD inputs, like from an iPhone Pro?

**Comment by wangpeng000** [comment]
Thank you for your prompt attention to our work! Unfortunately, we do not have access to an iPhone Pro device. However, we tested MBA-SLAM using real sequences captured by a RealSense camera. Although the RGB and depth data quality from the RealSense camera is not ideal, MBA-SLAM performed well.

**Comment by cdcseacave** [comment]
Indeed, but I'm afraid the quality of iPhone RGBD images is even worse. May I send you a dataset to test?

**Comment by wangpeng000** [comment]
@cdcseacave Of course, I’ll test your dataset this week. My email: wangpeng AT westlake.edu.cn

**Comment by cdcseacave** [comment]
pls try this: https://filetransfer.io/data-package/tLVLxsVS#link

**Comment by wangpeng000** [comment]
@cdcseacave Hi, we are currently processing your data due to an unmatched resolution, which will take some time. Additionally, the low-resolution depth data captured by the iPhone is really not so good, so the results may not turn out as expected.

**Comment by cdcseacave** [comment]
Indeed, that is why I was afraid too. Thank you for trying, looking forward to the results.
